Tensile/Source/lib/source/hip/HipSolutionAdapter.cpp

```diff
@@ -83,7 +84,20 @@ namespace Tensile
         {
             hipModule_t module;
 
-            HIP_CHECK_RETURN(hipModuleLoad(&module, path.c_str()));
+            // load file into buffer
+            std::ifstream coFile(path, std::ifstream::binary);
```

Comment:
You're missing a test to check if the ifstream is valid, this causes a crash in the Tensile tuning client

Reply:
I see, the client is only supposed to error only if _all_ code objects fail to load (although it looks like there's a bug in the implementation of this, but that's beside the point).

How does something like this sound?
```suggestion
            std::ifstream coFile(path, std::ifstream::binary);
            if (coFile) {
                // do the stuff
            }
            else
            {
                return hipErrorFileNotFound;
            }
```
I think this would mimic the previous behaviour

Reply:
That's exactly what I did to test it your changes, looks good to me!